### PR TITLE
fix - 2475 : Use aria-label and alt to  Improve accessibility

### PR DESCRIPTION
**Pull request by jeffin143**
Resolve https://github.com/opencollective/opencollective/issues/2475

Screenshot from 2019-11-15 16-19-07

### CODE REVIEW

File: components/StyledButton.js [hunk 1]
```diff
@@ -13,7 +14,9 @@ import StyledSpinner from './StyledSpinner';
  *
  * @see See [styled-system docs](https://github.com/jxnblk/styled-system/blob/master/docs/api.md) for usage of those props
  */
-const StyledButtonContent = styled.button`
+const StyledButtonContent = styled.button.attrs(({ ariaLabel }) => ({
+  'aria-label': ariaLabel,
+}))`
```
Comment (jeffin143) [inline components/StyledButton.js]: Although the problem is solved, But I am not sure how to add them as Props, I tired all possible way out, but couldn't resolve.

File: components/StyledButton.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (Betree) [inline components/StyledButton.js]: Oups, we can't have that

**Comment by Betree** [comment]
Removed unwanted code and rebased. Also changed the description to close https://github.com/opencollective/opencollective/issues/2475 when merged because it implements all the listed specs from the issue.

**Comment by Betree** [comment]
Thanks @jeffin143!

**Comment by jeffin143** [comment]
@Betree :) welcome 

Would love to contribute more, took some time off

**Pull request merged**